src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -29,6 +29,18 @@ To participate as a validator, a user must deposit 32 ETH into the deposit contr
 
 Whereas under proof-of-work, the timing of blocks is determined by the mining difficulty, in proof-of-stake, the tempo is fixed. Time in proof-of-stake Ethereum is divided into slots (12 seconds) and epochs (32 slots). One validator is randomly selected to be a block proposer in every slot. This validator is responsible for creating a new block and sending it out to other nodes on the network. Also in every slot, a committee of validators is randomly chosen, whose votes are used to determine the validity of the block being proposed.
 
+## How a Transaction Gets Executed in Ethereum PoS {#transaction-execution-ethereum-pos}
+
+The following provides an overview of how an account's transaction gets executed in Ethereum Proof of Stake, in contrast to [how transactions were mined](/developers/docs/consensus-mechanisms/pow/mining/) in Ethereum PoW.
+
+1. A user creates and signs a [transaction](/developers/docs/transactions/) with an [account's](/developers/docs/accounts/) private key. The user, usually assisted by their wallet, configures the transaction fees that they are prepared to pay in order to include the transaction in a block. These fees go partially to the validator ([tips](/developers/docs/gas/#priority-fee)) or [get burned](/developers/docs/intro-to-ether/#burning-ether) ([base fee](/developers/docs/gas/#base-fee)).
+2. The wallet sends the transaction to an Ethereum [execution client](/developers/docs/nodes-and-clients/) which verifies its validity.
+3. If the transaction is valid, the execution client adds it to its mempool and broadcasts it to other nodes, making it part of the public mempool. Advanced users may have sent their transaction instead to specialized and extended execution clients that hide the transaction from the public mempool, making it available to specialized block builders such as [Flashbots Auction](https://docs.flashbots.net/flashbots-auction/overview).
```

Comment:
I think it's a bit confusing to refer to the public mempool as there isn't really such a thing as a single coherent public transaction pool - it's just the collection of all the local mempools that each contain the transactions each node has heard about on the gossip network. 

```suggestion
3. If the transaction is valid, the execution client adds it to its local mempool (list of pending transactions) and also broadcasts it to other nodes over the execution layer gossip network. When other nodes hear about the transaction they add it to their local mempool too. Advanced users might refrain from broadcasting their transaction and insteads forward it to specialized block builders such as [Flashbots Auction](https://docs.flashbots.net/flashbots-auction/overview). This allows them to organize the transactions in upcoming blocks for maximum profit (MEV).
```